ui/src/components/forms/forms.scss

```diff
@@ -0,0 +1,26 @@
+.contact-form {
+label {
+  
+  margin: 10px;
+  padding: 5px;
+  span {
+    display:block;
+    
+  }
+  input {
+    width: 100%;
+  }
+}
```

Comment:
lets try to keep the spacing and indentation clean so bad habits dont get out of hand in the future